ParabolInc/parabol
# feat(Team Health): add new stage to retrospective

# Description

Fixes #7939
Add a new meeting stage to the retrospective stage.

## Demo

https://www.loom.com/share/44f85771297f4c459a93bc0ba5cb8ab5

## Testing scenarios

- add feature flag `teamHealth` to your organization
  ```graphql
  mutation AddOrgFeature {
    addFeatureFlagToOrg(orgIds: ["mDm2ilvXXO"], flag: teamHealth) {
      __typename
    }
  }
  ```
  - check retrospective settings
  - start a retro with team health check enabled and see the phase is present after the icebreaker
  - check again without icebreaker
  - start a retro with team health check disabled and see the phase is absent
